packages/components/src/color-control/index.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import { isFunction } from 'lodash';
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { withInstanceId } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import BaseControl from '../base-control';
+import ColorPicker from '../color-picker';
+
+const ColorControl = ( props ) => {
+	const { label, color, help, instanceId, className, onChange, disableAlpha } = props;
+	const id = `color-toggle-control-${ instanceId }`;
+
+	let describedBy, helpLabel;
+	if ( help ) {
+		describedBy = id + '__help';
+		helpLabel = isFunction( help ) ? help( color ) : help;
+	}
+	return (
+		<BaseControl
+			id={ id }
```

Comment:
With the BaseControls, aren’t the `id`s used to construct the `for` attribute for label accessibility? Not sure if that factors in here at all, just wanted to note in case there’s something else that needs to be added to support that.

Reply:
You're probably correct @chrisvanpatten however the linter tool fails for me if I don't provide an ID.